Podfile

```diff
@@ -12,6 +12,8 @@ target "Psiphon Browser" do
 	#pod "InAppSettingsKit", :path => "../InAppSettingsKit"
 	pod 'PsiphonClientCommonLibrary', :git => "https://github.com/Psiphon-Inc/psiphon-ios-client-common-library.git", :commit => '2e53a34'
```

Comment:
Not really relevant for this PR, but this pod entry is behind current https://github.com/Psiphon-Inc/psiphon-ios-client-common-library branch tip

Reply:
Fixed in https://github.com/Psiphon-Inc/endless/pull/161/commits/83005f8d4fe502c849dc58799c26a9a3ecd2751f.